lms/templates/enroll_staff.html

```diff
@@ -0,0 +1,41 @@
+<%page expression_filter="h" />
+<%inherit file="main.html" />
+<%namespace name='static' file='static_content.html'/>
+<%!
+    from django.utils.translation import ugettext as _
+    from courseware.courses import get_course_about_section
+%>
+
+<%block name="headextra">
+    <meta property="og:title" content="${course.display_name_with_default}"/>
+    <meta property="og:description" content="${get_course_about_section(request, course, 'short_description')}"/>
+</%block>
+
+<%block name="pagetitle">${course.display_name_with_default}</%block>
+
+<section class="course-info">
+    <header class="course-profile">
+        <div class="intro-inner-wrapper">
+            <div class="table">
+                <section class="intro">
+                    <div class="heading-group">
+                        <h3> ${_("You should Register before trying to access the Unit")}</h3>
+                    </div>
+                    <div class="heading-group">
+                        <p>
+                            ${course.display_name_with_default}
+                            <a href="#">${course.display_org_with_default}</a>
+                        </p>
+                    </div>
+                    <form role="form" id="enroll_staff_form" method="post" action="">
+                        <input type="hidden" name="csrfmiddlewaretoken" value="${ csrftoken }"/>
+                        <div class="main-cta">
+                            <input class="register" name="enroll" type="submit" value="${_('Enroll')}"/>
```

Comment:
Yes, these are the buttons. But I just follow the template pattern which is used in others templates. [Example](https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/course_about.html#L319)